kubernetes/delete-config-maps.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -e
 
+kubectl delete configmap nginx-json-errors    || true
```

Comment:
Add the file to existing config maps instead of creating a new one. It's an integral part of the current config, not something independent.

EDIT: I see it's in a different map because you can't put it in `/etc/nginx/`. You should explain that in a comment.

Maybe you could still put it in `/etc/` and just import using an absolute path?

Reply:
I put it in the same place as nginx-proxy `/ etc / nginx / snippets`, and in the ` default.conf` file, I used the absolute path to import it.

Reply:
I put it in the same place as nginx-proxy `/etc/nginx/snippets`, and in the ` default.conf` file, I used the absolute path to import it.

Reply:
You should call the map `nginx-config-snippets`.